### PR TITLE
Fix for Bug #124

### PR DESCRIPTION
**Pull request by kvi-ki**
Original => https://github.com/WomenWhoCode/london/issues/124 :
bug: On Mentor's bio buttons Show less, Show more displayed even if there is no text to hide #124
Expected result: there should not be a button "Show more"/"Show less" displayed where there is no need to show more text

Before:
Before

After: 
After

It's also could be done using .filter() and .map(), however, it doesn't look too different:

jQuery('.card-presentation')
.filter( index => jQuery('#card-text-'+index).prop("scrollHeight") <= jQuery('#card-text-'+index).prop("clientHeight"))
.map( () => jQuery('#btn-show-more-'+index).addClass(CLASS_HIDDEN));

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [✔] I checked the [contributor guide](https://github.com/WomenWhoCode/london/blob/main/CONTRIBUTING.md) 
- [✔] I checked include the related website screenshots changes

### CODE REVIEW

File: assets/js/mentors.js [hunk 1]
```diff
@@ -20,7 +20,14 @@ var controllerMentors = (function(jQuery) {
 
         jQuery("#presentation-"+index).removeClass(CLASS_HIDDEN);
         jQuery("#skills-"+index).addClass(CLASS_HIDDEN);
-        jQuery("#mentees-"+index).addClass(CLASS_HIDDEN);
+        jQuery("#mentees-"+index).addClass(CLASS_HIDDEN);   
+        
+        jQuery('.card-presentation')
+        .each( function(index) {
+            if(jQuery('#card-text-'+index).prop("scrollHeight") <= jQuery('#card-text-'+index).prop("clientHeight")){
```
Comment (Neha) [inline assets/js/mentors.js:27]: nitpick: We should  have these in a variable: 

`let cardHeight = jQuery('#card-text-'+index).prop("scrollHeight")`

`let clientHeight = jQuery('#card-text-'+index).prop("clientHeight")`

**Review by Neha — COMMENTED** [review]
LGTM. Added 1 small comment

**Review by nora-weisser — APPROVED** [review]

**Review by dricazenck — APPROVED** [review]
Thank you for the time and the solution. I think by fixing the comment we can merge it.

File: assets/js/mentors.js (hunk 1 above)
Comment (kvi-ki) [reply assets/js/mentors.js]: Yep, sure thing! Done

Comment (nora-weisser) [reply assets/js/mentors.js]: Thank you!

**Pull request merged**